javascript/src/main/pages/AnalyzeMessageData/AnalyzeReactions.js

```diff
@@ -1,7 +1,52 @@
-import React from "react";
+import React, {useState} from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import useSWR from "swr";
+import { fetchWithToken } from "main/utils/fetch";
+import MessageTableReaction from "main/components/ChannelMessageReaction/MessageTableReaction"
+import { Form, Button } from "react-bootstrap";
+
 
 const AnalyzeReactions = () => {
-    return <h1>Analyze Reactions</h1>
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const [ searchReaction, setSearchReaction ] = useState('');
+    const [ searchResults, setSearchResults ] = useState([]);
+    
+    const handleSearchReactionOnChange = (event) => {
+        setSearchReaction(event.target.value);
+    };
+    
+    const handleSearchReactionOnSubmit = () => {
+        var url = '';
```

Comment:
Can you use `let` instead of `var`?

Reply:
I still see `let` in the file.      As Andrew said: can you use `let` instead of var?  

That's better JavaScript style for current versions of JavaScript.    The reason is that `let` variables are properly lexically scoped, which means fewer unintended side effects, fewer weird bugs.

Please make this change!